lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -385,6 +385,10 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
   DXASSERT(opts.ExternalLib.empty() == opts.ExternalFn.empty(),
            "else flow above is incorrect");
 
+  for (const Arg *A : Args.filtered(OPT_precise_output)) {
+    opts.PreciseOutputs.emplace_back(A->getValue());
```

Comment:
You could just use:
`opts.PreciseOutputs = Args.getAllArgValues(OPT_precise_output);`
Instead of the loop.